src/components/date_picker/super_date_picker/super_date_picker.tsx

```diff
@@ -405,8 +414,8 @@ export class EuiSuperDatePickerInternal extends Component<
           className="euiDatePickerRange--inGroup"
           iconType={false}
           isCustom
-          startDateControl={<div />}
-          endDateControl={<div />}
+          startDateControl={<div onFocus={(e) => handleFocusActivity(e)} />}
+          endDateControl={<div onFocus={(e) => handleFocusActivity(e)} />}
```

Comment:
Static div's cannot take focus (without a tabIndex set) - so no need to pass onFocus here

```suggestion
          startDateControl={<div />}
          endDateControl={<div />}
```